Packs/Wiz/Integrations/Wiz/Wiz.yml

```diff
@@ -148,6 +148,13 @@ script:
       - LOW
       - INFORMATIONAL
       description: Get Issues of a specific severity and above.
+    - name: issue_type
+      auto: PREDEFINED
+      predefined:
+      - TOXIC_COMBINATION
+      - THREAT_DETECTION
+      - CLOUD_CONFIGURATION
+      description: Get Issues of a specific type.
```

Comment:
I think it should be 
- description (add the "-")

Reply:
I think the YAML is correct this way, I will check again and change if needed